src/Localizers.jl

```diff
@@ -89,6 +90,34 @@ http://arxiv.org/abs/2105.11341
 """
 struct SECFisher <: LocalizationMethod end
 
+
+"""
+    SECNice{FT <: Real} <: LocalizationMethod
+
+Sampling error correction as of Vishny, Morzfeld, et al. (2024).
```

Comment:
Can you please add the link to the preprint?

Reply:
Yes! Happy to now it's released!